public/src/actions/calls.jsx

```diff
@@ -1,18 +1,38 @@
 import axios from 'axios';
 
-import { SET_CALL_CURRENT, SET_CALL_NEXT, INCREMENT_CALLS, CLEAR_CALL_CURRENT, CLEAR_NEXT_CALL, CLEAR_COUNT_CALLS, PROMOTE_NEXT } from '../reducers/calls';
+import { SET_CALL_CURRENT,
+         CLEAR_CALL_CURRENT,
+         UPDATE_CALL_STATUS,
+         UPDATE_CALL_OUTCOME,
+         SET_CALL_CONTACT_INFO,
+         SET_CURRENT_CALL_ACTIVE,
+         SET_CURRENT_CALL_INACTIVE,
+         SET_VOLUNTEER_ACTIVE,
+         CLEAR_VOLUNTEER_ACTIVE } from '../reducers/calls';
 
-export function incrementCallCount() {
+export function setVolunteerActive() {
   return {
-    type: INCREMENT_CALLS
+    type: SET_VOLUNTEER_ACTIVE
   };
 }
-export function promoteNextToCurrent() {
+
+export function clearVolunteerActive() {
```

Comment:
This method should also DELETE to users/:id/camp/:id/calls to terminate the volunteer call?

Reply:
That's in the ticket I just did.